project/Dependencies.scala

```diff
@@ -34,8 +35,8 @@ object Dependencies {
   val googleDiff = "com.googlecode.java-diff-utils" % "diffutils" % googleDiffV
   val java8Compat = "org.scala-lang.modules" %% "scala-java8-compat" % java8CompatV
   val jgit = "org.eclipse.jgit" % "org.eclipse.jgit" % jgitV
-  val metaconfig = "com.geirsson" %% "metaconfig-typesafe-config" % metaconfigV
-  val metaconfigDoc = "com.geirsson" %% "metaconfig-docs" % metaconfigV
+  lazy val metaconfig = Def.setting("com.geirsson" %% "metaconfig-typesafe-config" % metaconfigV.value)
+  lazy val metaconfigDoc = Def.setting("com.geirsson" %% "metaconfig-docs" % metaconfigV.value)
```

Comment:
Having 2 settings in the middle of raw module ids is a bit weird - what about defining 2 modules and switching on the call-site instead? My experience of sbt is that `Dependencies` is better when it's as dumb and verbose as possible.

Reply:
ok!